internal/core/src/common/Json.h

```diff
@@ -263,6 +263,78 @@ class Json {
         return doc().at_pointer(pointer).get<T>();
     }
 
+    value_result<std::string>
+    at_string_any(std::string_view pointer) const {
+        AssertInfo(!pointer.empty(),
+                   "pointer must not be empty for at_string_any");
+        if (data_.size() == 0) {
+            return std::string{};
```

Comment:
we should distinguish between a field that does not exist and a field's value is empty string